scripts/resources_java_all.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
This is identical to `resources_go_all.sh` except `s/go/java/`.

How about refactoring into one script and run as `resources_all.sh go` or `resources_all.sh java`?